phonon/process.py

```diff
@@ -66,12 +67,13 @@ def __exit__(self, type, value, traceback):
     class AlreadyLocked(PhononError):
         pass
 
-    def __init__(self, process_ttl=int(0.5 * TTL), host='localhost', port=6379, db=1, heartbeat_interval=10, recover_failed_processes=True):
+    def __init__(self, process_ttl=int(0.5 * TTL), hosts=None, port=6379, db=1, heartbeat_interval=10, recover_failed_processes=True):
```

Comment:
Since this would be a breaking change for anybody who updates to the latest version of `phonon`, it might be worth keeping `host` as an acceptable parameter which gets converted to a `hosts` list with a deprecation warning?

Also, while in most cases it's likely that each host will be running redis on the same port, conceivably the port might differ from host to host. It may be worth allowing a way to specify ports on a per host basis?